.github/ISSUE_TEMPLATE/task.md

```diff
@@ -1,25 +1,27 @@
 ---
 name: Task
-about: Create a new task for the Innovation team to work on
+about: Create a new task for the Run team to work on
 title: ''
 assignees: ''
-
 ---
-**Context**
-*Describe / explain why we should do this: motivations, context or other info. Keep it brief and simple - PM*
 
+**Task Overview**
+_Provide a concise description of the task, explaining what needs to be done, why it's important, and its impact on the project or user experience._
```

Comment:
everything is written in the same line when i view the markdown file. is it wanted ?